proto/storage/compliance_operator_v2.proto

```diff
@@ -141,7 +144,7 @@ message ScanStatus {
 // Scan object per cluster
 message ComplianceOperatorScanV2 {
     string                     id                 = 1 [(gogoproto.moretags) = 'sql:"pk"'];
-    string                     scan_name          = 2 [(gogoproto.moretags) = 'search:"Compliance Scan Name,hidden,store" sql:"fk(ComplianceOperatorScanSettingV2:scan_name),no-fk-constraint,index=category:unique;name:scan_unique_indicator"'];
+    string                     scan_config_id     = 2 [(gogoproto.moretags) = 'search:"Compliance Scan Name,hidden,store" sql:"fk(ComplianceOperatorScanConfigurationV2:id),no-fk-constraint,index=category:unique;name:scan_unique_indicator"'];
```

Comment:
Again I think the postgres table for `ComplianceOperatorScanV2` might already be created for customers on 4.2 . Would this be a breaking change? The table would be completely empty but I don't know how the schema changes are applied on upgrade when a field's fk constraint is changed.

Reply:
For preservation, we discussed this externally.  This is not an issue:

Regarding the tables in customer environments, that isn't really a concern for 2 reasons:
I refactored the name too so if they had such a table it would be orphaned.
But most importantly for your awareness, if you pass the feature flag to the gen.go it will only apply the schema if the flag is on.
```//go:generate pg-table-bindings-wrapper --type=storage.ComplianceOperatorScanConfigurationV2 --search-category COMPLIANCE_SCAN_SETTINGS --references=storage.ComplianceOperatorProfileV2 --get-all-func --feature-flag ComplianceEnhancements```